src/pages/edge/edge-network/tutorials/datastream-config-override-rules.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Datastream config override for Edge events using Rules.
+description: This guide provides instructions on overriding datastream ID and datastream configuration for Edge Network Requests that are not initiated by the sendEvent API, all accomplished through Tags Rules.
+keywords:
+- Edge Network
+- Datastream config overrides
+- Datastream ID override
+- Guide
+---
+
+# Datastream config override for Edge events using Rules
+
+By default, all **Edge Experience events** are sent to the datastream configuration specified in the **Adobe Experience Platform Edge Network** extension configuration settings. This tutorial will offer examples demonstrating how to customize this configuration on a per-event basis
```

Comment:
- repetition on config, you may remove one of the occurrences: `.. the datastream specified in the ..`
- Use present tense and simplify: `This tutorial offers examples on how to customize ...`

Reply:
small repetition on examples + demonstrating, you may simplify like `examples on how to ...`